app/assets/javascripts/autogrow.js

```diff
@@ -26,16 +26,16 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-module.exports = function() {
-  return {
-    restrict: 'A',
-    scope: {
-      activity: '=',
-      commentInEdit: '='
-    },
-    templateUrl: '/templates/work_packages/tabs/_editable_comment.html',
-    link: function(scope){
+(function($) {
+  $(function() {
+    $('.-autogrow').autogrow({
+      onInitialize: true
+    });
```

Comment:
Is it possible to always add a newline just like it can be done with https://github.com/monospaced/angular-elastic?

When typing a long text (not hitting enter before the wrap happens) the animation is not really smoth. This is probably because of the one thread limitation browsers have. Always having a newline ready should make it nicer to use.

Additionally, as a user, I don't know what will happen until I get past the wrap, so I might be tempted to increase the height unnecessarily.